### ISSUE TITLE
is:open

### ISSUE THREAD

**Issue by extrablind** [symfony/panther#533]

**Comment by domis86** [comment]
Can you try adding another domain like :
```
'external_base_uri' => 'https://'.$domain.'.dev.test:4433',
```
which points to another front controller like `index_test.php` (copied from `index.php` but with changed `APP_ENV` to `test`)
?

**Comment by extrablind** [comment]
Good point on first solution which seems convenient but requires  extra processing to deploy (like deleting the `index_test.php` file). Basically symfony env the old way.
Just adding `index_test.php` with good conf and call `domain.dev:4433/index_test.php` instead of `domain.dev:4433` is ok as well and uses .env.test file
The second solution is interesting too, but I prefere to rely on `.env` file in order to keep it clean.

```php
# public/index_test.php
use App\Kernel;

require_once dirname(__DIR__).'/vendor/autoload_runtime.php';

return function (array $context) {
    $debug = true;
    return new Kernel('test',  $debug);
};
```

This will do the trick for the moment...
Thanks for your reply domis86

**Comment by arderyp** [comment]
i too can run `PANTHER_NO_HEADLESS=1 ./vendor/bin/phpunit`

but when I put the following in `phpunit.xml` it doesn't work:  `<env name="PANTHER_NO_HEADLESS" value="1"/>`
